app/views/sections/_form.html.erb

```diff
@@ -7,6 +7,12 @@
       <%= label_tag :number %>
       <%= form.number_field :number, in: 1...9999, :id => "number" , :placeholder => "000" , :class => "form-control" %>
 
+      <% if is_admin_html %>
```

Comment:
We need a default "none" or "no selection" or whatever in the dropdown. If the admin wants to edit a section, they have to choose a instructor.